app/assets/javascripts/dashboard/dashboard.js

```diff
@@ -565,6 +606,22 @@ $('body').on('click', '.accept-minion', function(e) {
   checkAcceptAllAvailability();
 });
 
+$('body').on('click', '.remove-minion', function(e) {
+  var minionId = $(this).data('minion-id');
+  var fqdn = $(this).data('minion-fqdn')
+
+  e.preventDefault();
+  deleteNode(minionId, fqdn);
+});
+
+$('body').on('click', '.reject-minion', function(e) {
+  var minionId = $(this).data('minion-id');
+  var fqdn = $(this).data('minion-fqdn')
+
+  e.preventDefault();
+  rejectNode(minionId, fqdn);
```

Comment:
Unfortunately we don't have access to the minion hostname/fqdn before accepting it, so in this case you can simply use the minion id.